dscreator/sources/ferrybox/queries.py

```diff
@@ -34,7 +34,6 @@ def get_track(
         track.uuid = :track_uuid
         AND track.time > :start_time
         AND track.time <= :end_time
-        AND track.qc = 1
```

Comment:
Shouldn't this be there? this would add points that does not look valid

Reply:
I disabled it for testing. Im not sure which data Andrew has pulled and wanted to check. If you require qc=1 many data points will not have location. 

There are different opinions how to handle noisy gps. Some think that these data could be publish because you can interpolate noisy points.

Reply:
Reverted to good tracks only.